early-return/activity.go

```diff
@@ -0,0 +1,33 @@
+package earlyreturn
+
+import (
+	"context"
+	"errors"
+
+	"go.temporal.io/sdk/activity"
+)
+
+func InitTransaction(ctx context.Context, transactionId, fromAccount, toAccount string, amount float64) error {
+	logger := activity.GetLogger(ctx)
+	if fromAccount == "" {
+		return errors.New("invalid fromAccount")
+	}
+	if toAccount == "" {
+		return errors.New("invalid toAccount")
+	}
+	if amount == 0 {
+		return errors.New("invalid amount")
+	}
+	logger.Info("Transaction initialized")
+	return nil
+}
+
+func CancelTransaction(ctx context.Context, transactionId string) {
```

Comment:
nit: could add some delay here and to `CompleteTransaction` so "early return" is returning an observable amount before the real workflow

Reply:
👍 Good idea